### PR TITLE
Added django-extensions (shell_plus) req.

### PR DESCRIPTION
**Pull request by ralphiee22**
## Summary

Added django-extensions to dev requirements. Allows us to preload django models using `kolibri manage shell_plus`.

## TODO

- [x] New dependencies (if any) added to requirements file

## Documentation
http://django-extensions.readthedocs.io/en/latest/shell_plus.html

## Screenshots (if appropriate)

**_SO COOL!_**

screen shot 2016-09-17 at 1 41 42 pm

### CODE REVIEW

**Comment by ralphiee22** [comment]
@rtibbles @benjaoming It seems we need to put `django_extensions` into `INSTALLED_APPS`. But I'm guessing this also means we have to bundle it with kolibri, which I doubt we want to do. Any way to compromise?

**Comment by rtibbles** [comment]
Probably best to make a dev settings file that imports base settings. We can then use those to run with the --settings flag.

File: kolibri/utils/cli.py [hunk 1]
```diff
@@ -62,6 +62,7 @@
   kolibri url               Tell me the address of Kolibri
   kolibri shell             Display a Django shell
   kolibri manage help       Show the Django management usage dialogue
+  kolibri manage shell_plus Preload all models in Django shell
```
Comment (benjaoming) [inline kolibri/utils/cli.py:65]: It's not nice to load our CLI usage instructions any further than they already are. `shell_plus` is a developer tool that some developers may like, but only some, so basically I don't think it's warranted in the CLI.

**Comment by benjaoming** [comment]
Dev settings and dev dependencies have to be separated from what we distribute. On one hand, that's why we have `requirements_dev.txt`, but additionally we don't let them overflow into the main application (as with the CLI usage instructions in this PR).

The KA Lite codebase is heavily polluted with development tools, even a bundled version of Django ended up being hacked (in order to introduce a developer utility!). Lesson learned :)

I recommend that you create your own `dev.sh` script with your own favorite settings. If you want `django_extensions`, you can add it to `INSTALLED_APPS` in a local `settings.py` file.

I would add: It's nice to share tips about cool tools, but it's also sort of metawork and adding complexity to our codebase for the sake of developer setups which may be highly customized and attract a lot of discussion.

Using a basic lowest common denominator (the Django defaults) will help us to easily setup our own favorite environments.

If you have problems integrating developer utilities, then that's an issue that should be solved of course :)

**Comment by benjaoming** [comment]
Btw. `pip install bpython` will turn `kolibri shell` into a fancy shell without any further configuration :)

**Comment by ralphiee22** [comment]
Closing this PR because I will use my own customization.

**Pull request closed without merging**